src/config.rs

```diff
@@ -97,7 +99,7 @@ impl ConfigDeserializer {
 
         match deser_result {
             toml::value::Value::Integer(threads) => {
-                if threads > 0 && threads <= MAX_THREADS {
+                if threads > 0 && threads <= i64::from(MAX_THREADS) {
```

Comment:
As I told you before, I think it's better to cast here, but not sure about that. Why are we changing the `MAX_THREADS` constant?

Reply:
Well, I don't expect `MAX_THREADS` to ever be 1) negative or 2) over 256, so `u8` is good enough. We still have to take care of `::Integer` returning `i64` though. (Also: Clippy told me this was preferred)